src/pkg/scan/dao/scanner/registration.go

```diff
@@ -121,14 +121,6 @@ func ListRegistrations(ctx context.Context, query *q.Query) ([]*Registration, er
 	if err != nil {
 		return nil, err
 	}
-
-	// Order the list
-	if query.Sorting != "" {
-		qs = qs.OrderBy(query.Sorting)
-	} else {
-		qs = qs.OrderBy("-is_default", "-create_time")
```

Comment:
Please check if we need to keep this line to ensure backward compatibility without introducing inconsistent results from previous versions, because the common query logic processor will not handle this special case.